docs/user-guide/multiple_sources.md

```diff
@@ -0,0 +1,62 @@
+# Multiple Sources for an Application
+
+Argo CD has the ability to specify multiple sources to add services to the Application. Argo CD compiles all the sources and reconciles each source individually for creating the application.
+
+You can provide multiple sources using the `sources` field. When you specify the sources field, Argo CD will ignore the values under `source` field for generating the application.
```

Comment:
Can you add the `sources` field to the docs/operator-manual/application.yaml example file?